spec/minimax_server/core_spec.clj

```diff
@@ -1,8 +1,8 @@
 (ns minimax-server.core-spec
-  (import [scarvill.httpserver.response ResponseBuilder])
-  (require [speclj.core :refer :all]
+  (:require [speclj.core :refer :all]
             [minimax-server.core :refer :all]))
 
-(describe "temporary tests"
-  (it "can use things from http server jar"
-    (should-be-a ResponseBuilder (new ResponseBuilder))))
+(describe "main"
```

Comment:
This was actually a test to make sure that we can import the server jar and use Spencer's server. But I understand this isn't really needed because the compiler will throw an error if it cannot reference a class from the server. I have deleted this file.